app/app/urls.py

```diff
@@ -402,6 +405,11 @@
     # gitcoinbot
     url(settings.GITHUB_EVENT_HOOK_URL, gitcoinbot.views.payload, name='payload'),
     url(r'^impersonate/', include('impersonate.urls')),
+
+    # wagtail
+    re_path(r'^cms/', include(wagtailadmin_urls)),
```

Comment:
@pinkiebell 
two questions:

- is there any public facing index of all of the helpdesk items we can use?
- is there any way to get the helpdesk items into the sitemap (see sitemaps.py)

Reply:
@owocki
1. That would have to be built. It is also possible to redirect to a custom page in the meantime.
2. Wagtail has a sitemap module; that should work.